ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -47,12 +47,23 @@ packet)
     # shellcheck source=/dev/null
     source "${SHARED_DIR}/ds-vars.conf"
     copy_test_binaries
-    export TEST_PROVIDER=\"\"
+
+    # Currently all v6 deployments are disconnected, so we have to tell
+    # openshift-tests to exclude those tests that require internet
+    # access.
+    if [[ "$IP_STACK" -eq "v6" ]];
```

Comment:
I don't think `IP_STACK` it's available in [`ds-vars.conf`](https://github.com/openshift/release/blob/fa63247905df5522e3271f5f7d6d39aea9e6a65d/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L102-L104), it must be added previously there in the `devscript-setup` step

Reply:
Oh thanks, good catch